CHANGELOG.rst

```diff
@@ -10,6 +10,10 @@ Features
 - Added mean error
   :py:func:`~xskillscore.me`. (:issue:`202`, :pr:`200`)
   `Andrew Huang`_
+- :py:func:`~xskillscore.brier_score` and :py:func:`~xskillscore.rps` now contain
+  keyword ``fair`` to account for ensemble-size adjustments, but defaults to ``False``.
+  Adjustment not implemented yet for :py:func:`~xskillscore.crps_ensemble`. PR welcome.
```

Comment:
can remove this line. Moved to an issue: https://github.com/xarray-contrib/xskillscore/issues/260

Reply:
I thought this would be a nice advertisement for the PR needed. issues quickly pileup and nobody sees them. but I can remove that line also

Reply:
now removed